pkgs/applications/blockchains/trezor-suite/default.nix

```diff
@@ -53,6 +53,10 @@ appimageTools.wrapType2 rec {
   '';
 
   meta = with lib; {
+    # this application does not perform its core functionality.
```

Comment:
```suggestion
    # trezor-suite fails to see a connected hardware wallet
```

Reply:
@sudoforge yes, please rework the message to be more specific